### PR TITLE
Javascript javascript1 week2

### PR DESCRIPTION
**Pull request by sumanghimire79**

### CODE REVIEW

File: javascript/javascript1/week2/weather-wear/weatherWear.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+let todaysDress;
+if (temperature>17){
+    todaysDress= "It is a a bit warm today so it is adviced to wear shorts and a t-shirt";
+} else {
+    todaysDress = "it is a bit colder today so it is adviced to  wear a jacket";
+}
+    return todaysDress;
+}
+const clothesToWear = dressAdviser(13);
+console.log( clothesToWear);
```
Comment (esteficodes) [inline javascript/javascript1/week2/weather-wear/weatherWear.js:14]: Great! 😀

**Review by esteficodes — APPROVED** [review]
Good job! Check the comments to add the missing details and you'll be all set to merge.🎈

File: javascript/javascript1/week2/student-manager/studentManager.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+addStudentToClass("Queen");// array.length did not increase
+console.log(class07Students);
+
+addStudentToClass("MR. Langsome");
+
+console.log(class07Students);
+addStudentToClass("Queen"); // assigining queen again?
+
+getNumberOfStudents();
```
[the remark this answers is not in the document]
Comment (amritpandey) [reply javascript/javascript1/week2/student-manager/studentManager.js:59]: what a clear presentation of your code. I really like it. I am gonna follow your way.
 And I think you forgot to push the queen to the array. you can simply add  class07Students.push(studentName); in line no 16.

File: javascript/javascript1/week2/formal-fullname/formalFullName.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+        }else {
+            fullName = fullname1 + fullname2 ;
+        }
+        return fullName;
+    }
+    var myFullName = getFullname("Suman", " Ghimire", true);
+    console.log (myFullName);
+    var mySonsFullName = getFullname("Aayam", " Ghimire", false);
+    console.log (mySonsFullName);
```
[the remark this answers is not in the document]
Comment (sumanghimire79) [reply javascript/javascript1/week2/formal-fullname/formalFullName.js:31]: yes, i have added the code !

File: javascript/javascript1/week2/student-manager/studentManager.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (sumanghimire79) [reply javascript/javascript1/week2/student-manager/studentManager.js:59]: thank you so much both of you ! i have pushed the Queen to the array to get the .length increased. 
Dear Amrit we learn form each other. i eye through your code,  it is true that your code is atomic  and lies in next level. most importantly you involve in interaction ! that is great !

File: javascript/javascript1/week2/weather-wear/weatherWear.js (hunk 1 above)
Comment (sumanghimire79) [reply javascript/javascript1/week2/weather-wear/weatherWear.js:14]: yes , agreed !

File: javascript/javascript1/week2/formal-fullname/formalFullName.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (sumanghimire79) [reply javascript/javascript1/week2/formal-fullname/formalFullName.js:31]: i checked all four possible scenarios considering 'formalFullName' and 'gender' ! all those passes ! i think it should be ok but still thinking there must be another smarter way of doing it . please have a look one time again in your convenience !

**Pull request merged**